packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -121,13 +126,38 @@ contract DeployOwnership is Deploy {
         console.log("New LivenessModule deployed at %s", address(addr_));
     }
 
+    /// @notice Deploy a DeputyGuardianModule for use on the Security Council Safe.
+    ///         Note this function does not have the broadcast modifier.
+    function deployDeputyGuardianModule() public returns (address addr_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        SuperchainConfig superchainConfig = SuperchainConfig(makeAddr("SuperchainConfig"));
+        addr_ = address(
+            new DeputyGuardianModule({
+                _safe: councilSafe,
+                _superchainConfig: superchainConfig,
+                _deputyGuardian: mustGetAddress("FoundationSafe")
+            })
+        );
+
+        save("DeputyGuardianModule", addr_);
+        console.log("New DeputyGuardianModule deployed at %s", addr_);
+    }
+
     /// @notice Deploy a Security Council with LivenessModule and LivenessGuard.
     function deployAndConfigueSecurityCouncilSafe() public returns (address addr_) {
         Safe safe = Safe(payable(deploySafe("SecurityCouncilSafe")));
 
+        vm.startBroadcast();
+        address deputyGuardianModule = deployDeputyGuardianModule();
```

Comment:
`deployDeputyGuardianModule` uses `makeAddr("SuperchainConfig")`—which is not a real superchain config—as the superchain config address for the broadcast. I think it should instead get the address of what `deploySuperchainConfig` deployed?

Reply:
That works too, can do